### PR TITLE
chore: fix ci

### PR DESCRIPTION
**Pull request by tony-go**
Tryin to work on a new feature (check version). I figured that test was broken on master but green on the CI.

TODO:

- [x] fix ci
- [x] fix tests

### CODE REVIEW

**Comment by targos** [comment]
Just curious, what CI failure are you trying to fix? GitHub actions are green on master.

**Comment by tony-go** [comment]
> Just curious, what CI failure are you trying to fix? GitHub actions are green on master.

It's green in the CI but tests within fail...

**Comment by tony-go** [comment]
Now, that CI fails as expected, I'll attempt to fix tests now 😅

**Comment by targos** [comment]
> > Just curious, what CI failure are you trying to fix? GitHub actions are green on master.
> 
> It's green in the CI but tests within fail...

haha no need for the [volkswagen](https://www.npmjs.com/package/volkswagen) module in this project :D

**Comment by tony-go** [comment]
Friendly ping @fraxken

File: test/commands/summary.spec.js [hunk 1]
```diff
@@ -26,7 +26,6 @@ test("summary should execute summary command on fixtures 'result-test1.json'", a
     /\(48\) MIT - \(1\) BSD-3-Clause - \(3\) ISC.*$/,
     /.*/
   ];
-  tape.plan(lines.length * 2);
```
Comment (targos) [inline test/commands/summary.spec.js:29]: This plan seemed correct.

Comment (tony-go) [reply test/commands/summary.spec.js:29]: Finally, catch the error. The `@nodesecure/scanner` changed a bit and the fixture was simply outdated.

**Pull request merged**